### ISSUE TITLE
Claim: Tenant Expulsion

### ISSUE THREAD

**Issue by arekmano** [Cyberjusticelab/JusticeAI#184]
*Description*
As a user, I would like the system to predict whether or not tenant expulsion is justified

*Scope of Work*
- [x] Create supervised learning algorithm benchmark for evaluating predictions
- [x] Implement supervised learning algorithm that can evaluate whether or not tenant expulsion is justified based on facts present in the case .
- [x] Extract using regex, or similar technique whether or not the tenant was expulsed in a specific case.  

*Demo requirement*
1. The user selects the facts that are relevant to their case
2. The user sends the facts to the system
3. The system displays "Tenant expulsion is justified" or "Tenant expulsion is not justified" 

*Acceptance Criteria*
- The user is able perform demo steps
- The F1 score of the supervised learning algorithm is >50%.

**Documentation, files, diagrams, etc. relating to this feature:**
Url, books, etc.

***Changes***:
- [Optional]

**Comment by ghost** [comment]
This is covered by lease termination.

**Comment by arekmano** [comment]
| Prediction | f1|
| -----------|-----|
| additional_indemnity_date | [ 0.87275506  0.87743835]|
| additional_indemnity_money | [ 0.87868732  0.74466434]|
| declares_housing_inhabitable | [ 0.999875  0.      ]|
| declares_resiliation_is_correct | [ 0.95337609  0.69552941]|
| orders_expulsion | [ 0.97162192  0.9619883 ]|
| orders_immediate_execution | [ 0.91473297  0.81991443]|
| orders_resiliation | [ 0.97318436  0.9659671 ]|
| orders_tenant_pay_first_of_month | [ 0.99673981  0.        ]|
| rejects_landlord_demand | [ 0.98689956  0.5255814 ]|
| rejects_tenant_demand | [ 0.98617865  0.42245989]|
| tenant_ordered_to_pay_landlord | [ 0.72910864  0.68399675]|
| tenant_ordered_to_pay_landlord_legal_fees | [ 0.80152357  0.69338422]|

**Comment by Samuel-Campbell** [comment]
it's a different category @mihaiqc .

**Comment by Vynny** [comment]
Was not integrated with NLP. Need to go over fact specifics with Sam prior to integration.

**Comment by ghost** [comment]
@Samuel-Campbell I understand, but contextually speaking it shouldn't be.

The outcome is the same. You can't NOT be expelled out of the apartment if your lease is terminated in court.

So I don't understand the purpose of this category. It makes little sense to me and I have no idea how to differentiate it from lease termination in the problem category classifier.

**Comment by Vynny** [comment]
This will likely not require any additional work once #186 is merged. The changes in that PR will make it so that any outcome returned from a prediction will be shown to the user. 

As I understand it this is simply an add-on outcome, thus it is already covered by the "orders_resiliation" outcome being mapped to the "lease_termination" claim category.
